code/src/cljs/sixsq/nuvla/ui/plugins/full_text_search.cljs

```diff
@@ -28,19 +29,33 @@
        :fx [[:dispatch change-event]]})))
 
 (defn FullTextSearch
-  [{:keys [db-path change-event] :as opts}]
+  [{:keys [db-path change-event placeholder-suffix] :as opts}]
   (dispatch [::helpers/set db-path ::change-event change-event])
   (let [tr   @(subscribe [::i18n-subs/tr])
+        search-placeholder (tr [:search])
         text @(subscribe [::helpers/retrieve db-path ::text])]
     [ui/Input
      (-> opts
-         (dissoc :db-path :change-event)
-         (assoc :placeholder (tr [:search])
+         (dissoc :db-path :change-event :placeholder-suffix)
+         (assoc :placeholder (if placeholder-suffix
```

Comment:
I think it's better to keep this component simple. 
Add a placeholder option and use it if passed or we use the default value. No logic.

Reply:
So the logic will be passed by the page using the component to update the placeholder option

Reply:
`FullTextSearch` is dumber again! Makes sense having less control flow in there.

I changed the dictionary for `:search` and stripped out the `...` from it, which means we have to append it here and at the other places we used it.

There were three other usages but I changed only two of those to append `...` because those two were `placeholder` attributes.

The third one was the displayed value of button (on API page) where I think it's better without the `...`.

What do you think?